configs/fcaf3d/fcaf3d.py

```diff
@@ -0,0 +1,7 @@
+_base_ = ['../_base_/models/fcaf3d.py', '../_base_/default_runtime.py']
```

Comment:
We may do not need a base config in the fcaf3d folder.
Would better to have one for a specific dataset and others inherit from it